CONTRIBUTING.md

```diff
@@ -112,6 +113,27 @@ Then verify the new Rust bindings compile and pass some basic tests:
 $ cargo test -p tests_expectations
 ```
 
+## Generating graphviz dot file
```

Comment:
Nittiest of nit picks, but: **T**his **S**hould **B**e in **T**itle **C**ase. Also s/file/files/.

> Generating Graphviz Dot Files